vms/platformvm/network/network.go

```diff
@@ -30,8 +30,6 @@ type Network interface {
 
 	// IssueTx verifies the transaction at the currently preferred state, adds
 	// it to the mempool, and gossips it to the network.
-	//
-	// Invariant: Assumes the context lock is held.
```

Comment:
Should we document that the lock is assumed _not_ to be held?

Reply:
I think we should ^

Reply:
I've always felt really weird documenting anything along the lines of assumes lock is held/not held since if a lock is being grabbed it implies that the code assumes the lock isn't held and vice-versa

Reply:
This will be irrelevant with the next PR anyways... so I think it's fine